sormas-backend/src/test/java/de/symeda/sormas/backend/docgeneration/TemplateTestUtil.java

```diff
@@ -51,4 +51,13 @@ public static void writeJsonProperty(Object object) throws JsonProcessingExcepti
 	public static String cleanLineSeparators(String text) {
 		return text.replaceAll("\\r\\n?", "\n");
 	}
+
+	public static String addOSBasedLineSeparators(String text) {
```

Comment:
```suggestion
	public static String updateLineSeparatorsBasedOnOS(String text) {
```

Because the method doesn't add line separators, but merely replaces them (if at all), "add" is not a good method name here